system/Session/Session.php

```diff
@@ -145,31 +218,23 @@ public function initialize()
 
 		// Is session ID auto-regeneration configured? (ignoring ajax requests)
 		if ((empty($_SERVER['HTTP_X_REQUESTED_WITH']) ||
-		     strtolower($_SERVER['HTTP_X_REQUESTED_WITH']) !== 'xmlhttprequest')
-		    && ($regenerate_time = $this->sessionTimeToUpdate) > 0
+				strtolower($_SERVER['HTTP_X_REQUESTED_WITH']) !== 'xmlhttprequest') && ($regenerate_time = $this->sessionTimeToUpdate) > 0
 		)
 		{
-			if ( ! isset($_SESSION['__ci_last_regenerate']))
+			if (!isset($_SESSION['__ci_last_regenerate']))
 			{
 				$_SESSION['__ci_last_regenerate'] = time();
-			}
-			elseif ($_SESSION['__ci_last_regenerate'] < (time() - $regenerate_time))
+			} elseif ($_SESSION['__ci_last_regenerate'] < (time() - $regenerate_time))
 			{
-				$this->regenerate((bool)$this->sessionRegenerateDestroy);
+				$this->regenerate((bool) $this->sessionRegenerateDestroy);
 			}
 		}
 		// Another work-around ... PHP doesn't seem to send the session cookie
 		// unless it is being currently created or regenerated
 		elseif (isset($_COOKIE[$this->sessionCookieName]) && $_COOKIE[$this->sessionCookieName] === session_id())
 		{
 			setcookie(
-				$this->sessionCookieName,
-				session_id(),
-				(empty($this->sessionExpiration) ? 0 : time() + $this->sessionExpiration),
-				$this->cookiePath,
-				$this->cookieDomain,
-				$this->cookieSecure,
-				true
+					$this->sessionCookieName, session_id(), (empty($this->sessionExpiration) ? 0 : time() + $this->sessionExpiration), $this->cookiePath, $this->cookieDomain, $this->cookieSecure, true
```

Comment:
Personally, I find this line easier to read in the old formatting, with the many options on their own lines, but it's probably fine either way.